CONTRIBUTING.md

```
@@ -1,96 +1,197 @@
 
-### Linting and formatting
+---
 
-The project uses [`@antfu/eslint-config`](https://github.com/antfu/eslint-config) for linting and formatting. Run before submitting a PR:
+## Repository Structure
 
-```bash
-npm run lint        # Lint
-npm run lint:fix    # Lint and auto-fix
-npm run typecheck   # Type-check with vue-tsc
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)